test.ts

```diff
@@ -113,7 +113,7 @@ Deno.test({
     assertEquals(graph.modules.length, 1);
     assertEquals(
       graph.modules[0].error,
-      "load rejected or errored\n  Specifier: file:///a/test.ts",
```

Comment:
I added this in a recent PR, but when I integrated it into the CLI it was too noisy and we already display this info.